src/style.css

```diff
@@ -1,3 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  max-width: 1200px;
+  min-width: 1024px;
+  padding: 0 50px;
+  /* text-align: right; */
```

Comment:
Consider removing redundant comments to keep your code clean. You don't use this line and it doesn't have a meaning.